providers/summary/clearlydefined.js

```diff
@@ -100,6 +101,14 @@ class ClearlyDescribedSummarizer {
     setIfValue(result, 'described.releaseDate', extractDate(data.releaseDate))
     setIfValue(result, 'licensed.declared', data.declaredLicense)
   }
+
+  _extractLicenseFromFiles(files) {
+    if (!files) return
+    for (const file of files) {
+      if (file.license && file.license != 'NOASSERTION')
+        return file.license
```

Comment:
Is this a place where we might form an SPDX expression based on multiple files having different licenses?